src/hosted-form/hosted-form.ts

```diff
@@ -63,7 +67,14 @@ export default class HostedForm {
         return await this._getFirstField().submitForm(
```

Comment:
Make sure `async/await` is consistent (don't use a mixture of `then/catch` and `async/await`)

```ts
try {
    return await this._getFirstField().submitForm(
        this._fields.map(field => field.getType()), 
        this._payloadTransformer.transform(payload)
    );
} catch (error) {
    const additionalAction = await this._paymentHumanVerificationHandler.handle(error);

    return await this._getFirstField().submitForm(
        this._fields.map(field => field.getType()),
        this._payloadTransformer.transform(payload, additionalAction)
    );
}
```